### PR TITLE
Add VM_EXTRADISKS_LIST

### PR DESCRIPTION
**Pull request by lranjbar**
Adding VM_EXTRADISKS_LIST to be able to control the names and number of disks created using this option.

- [X] Add VM_EXTRADISKS_LIST as a variable
- [ ] Test adding VM_EXTRADISKS_SIZE as a variable
- [ ] Work through ci test errors

### CODE REVIEW

File: vm_setup_vars.yml [hunk 1]
```diff
@@ -15,12 +15,14 @@ flavors:
...
     vcpu: "{{ lookup('env', 'MASTER_VCPU') | default('8', true) }}"
     extradisks: "{{ lookup('env', 'VM_EXTRADISKS') | default(false) }}"
+    extradisks_list: "{{ lookup('env', 'VM_EXTRADISKS_LIST').split(' ') | default(['vdb']) }}"
 
   worker:
     memory: "{{ lookup('env', 'WORKER_MEMORY') | default('8192', true) }}"
     disk: "{{ lookup('env', 'WORKER_DISK') | default('20', true) }}"
     vcpu: "{{ lookup('env', 'WORKER_VCPU') | default('4', true) }}"
     extradisks: "{{ lookup('env', 'VM_EXTRADISKS') | default(false) }}"
+    extradisks_list: "{{ lookup('env', 'VM_EXTRADISKS_LIST').split(' ') | default(['vdb']) }}"
```
Comment (hardys) [inline vm_setup_vars.yml:25]: I think this needs to be a variable defined outside of the flavors block, see: 

https://github.com/metal3-io/metal3-dev-env/blob/master/vm-setup/roles/libvirt/defaults/main.yml#L24

vs extradisks which is defined per-flavor:

https://github.com/metal3-io/metal3-dev-env/blob/master/vm-setup/roles/common/defaults/main.yml#L22

Comment (lranjbar) [reply vm_setup_vars.yml]: Thanks for looking at it. This did fix the issue when I got around to testing it

**Comment by lranjbar** [comment]
Tested this change with dev-scripts in the lab and the second disk is showing now.

Here is what the volume pool looks like when putting in two disks:
```
 virsh vol-list oooq_pool
 Name                      Path
---------------------------------------------------------------------------
 ostest_master_0.qcow2     /home/dev-scripts/pool/ostest_master_0.qcow2
 ostest_master_0_vdb.img   /home/dev-scripts/pool/ostest_master_0_vdb.img
 ostest_master_0_vde.img   /home/dev-scripts/pool/ostest_master_0_vde.img
 ostest_master_1.qcow2     /home/dev-scripts/pool/ostest_master_1.qcow2
 ostest_master_1_vdb.img   /home/dev-scripts/pool/ostest_master_1_vdb.img
 ostest_master_1_vde.img   /home/dev-scripts/pool/ostest_master_1_vde.img
 ostest_master_2.qcow2     /home/dev-scripts/pool/ostest_master_2.qcow2
 ostest_master_2_vdb.img   /home/dev-scripts/pool/ostest_master_2_vdb.img
 ostest_master_2_vde.img   /home/dev-scripts/pool/ostest_master_2_vde.img
 ostest_worker_0.qcow2     /home/dev-scripts/pool/ostest_worker_0.qcow2
 ostest_worker_0_vdb.img   /home/dev-scripts/pool/ostest_worker_0_vdb.img
 ostest_worker_0_vde.img   /home/dev-scripts/pool/ostest_worker_0_vde.img
 ostest_worker_1.qcow2     /home/dev-scripts/pool/ostest_worker_1.qcow2
 ostest_worker_1_vdb.img   /home/dev-scripts/pool/ostest_worker_1_vdb.img
 ostest_worker_1_vde.img   /home/dev-scripts/pool/ostest_worker_1_vde.img
 ostest_worker_2.qcow2     /home/dev-scripts/pool/ostest_worker_2.qcow2
 ostest_worker_2_vdb.img   /home/dev-scripts/pool/ostest_worker_2_vdb.img
 ostest_worker_2_vde.img   /home/dev-scripts/pool/ostest_worker_2_vde.img
```

**Comment by stbenjam** [comment]
/ok-to-test
/approve

Do you maybe want to include the size option? There's also `extradisks_size`

**Comment by stbenjam** [comment]
Error looks related ` "stderr": "error: invalid argument: invalid path /opt/dev-scripts/pool/ostest_worker_1_.img not assigned to domain",�[0m`

File: config_example.sh [hunk 2]
```diff
@@ -197,6 +197,13 @@ set -x
 #export WORKER_DISK=20
 #export WORKER_VCPU=4
 
+# Add extradisks to VMs
+# export VM_EXTRADISKS=true
+
+# Configure how many extra disks to add to VMs. Takes a string of disk
+# names delimited by spaces. Example "vdb vde"
+# export VM_EXTRADISKS_LIST="vdb vde"
```
Comment (hardys) [inline config_example.sh:205]: nit, this example looks odd, wouldn't you generally expect contiguous device names, e.g `vdb vdc`?

Comment (lranjbar) [reply config_example.sh:205]: true, that will be better for an example. I sometimes flip "b" and "d" when I read these things 😅

**Comment by flaper87** [comment]
/close

@mkowalski took this commit and pushed it across 

Awesome work, you both!

**Pull request closed without merging**